src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,14 +1,34 @@
 package core.basesyntax.service;
 
+import core.basesyntax.RegistrationException;
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_LENGTH_PASSWORD = 6;
+    private static final int MIN_AGE_FOR_REGISTRATION = 18;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new RegistrationException(
+                    "User is null");
+        } else if (user.getId() == 0 || user.getLogin() == null || user.getPassword() == null) {
```

Comment:
```suggestion
        } else if (user.getLogin() == null || user.getPassword() == null) {
```
id може бути 0 тому що ми якраз зараз і створюємо юзера, генеруємо йому нове ід